mesh/tests/accounts_tests.py

```diff
@@ -31,17 +35,17 @@ def test_get_all_accounts(self):
         """
         Test case for getting all accounts. 
 
-        A GET request is sent to the '/api/accounts/' endpoint.
+        A GET request is sent to the '/api/accounts' endpoint.
         The test passes if the response status code is 200.
         """
-        response = self.client.get('/accounts/')
+        response = self.client.get('/accounts')
```

Comment:
On my end, this test only passes if I put `/accounts/`. Can you confirm if it passes on your end?

Reply:
yess you are right. For some reasons, it also accepts both `/accounts` when i run the test functions. But when I actually tried to send the request to the API, it only accepts the `/accounts/`. I changed the documentation to `/accounts/`